packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -72,17 +79,16 @@ const ProjectForm: FC<Props> = ({
             {showGeneralSettings && (
                 <SettingsGridCard>
                     <div>
-                        <H5>General settings</H5>
+                        <Title order={5}>General settings</Title>
                     </div>
 
                     <div>
-                        <Input
+                        <TextInput
```

Comment:
I believe this whole form is wrapped with react-form-hook library and previous import of `ReactHookForm/Input` is react-form-hook enabled component. So 99% sure this should not work. to test this, change project name, save, reload

Reply:
hii @IrakliJani , so should i use mantine form hooks in place of react-form-hook ?

Reply:
since this is a part of bigger change - we can use react hook form and it's hooks to get it working. it can be switched to mantine-form-hook after finishing whole project settings migration. 

this should be enough:
```
const { register } = useFormContext();
<MantineTextInput [props that you have] {...register("field name that was used before on Input field")} />
```

docs: https://www.react-hook-form.com/api/useformcontext/

Reply:
@jayanand05 let me know if that works or if you need help!

Reply:
@IrakliJani, I updated the code request you to look once?